src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -30,6 +30,8 @@
   "text.autoconfig.skyblocker.option.general.fishing.enableFishingHelper": "Enable Fishing Helper",
   "text.autoconfig.skyblocker.option.general.fairySouls": "Fairy Souls Helper",
   "text.autoconfig.skyblocker.option.general.fairySouls.enableFairySoulsHelper": "Enable Fairy Souls Helper",
+  "text.autoconfig.skyblocker.option.general.fairySouls.highlightFoundSouls": "Highlight found souls",
+  "text.autoconfig.skyblocker.option.general.fairySouls.highlightOnlyNearbySouls": "Only highlight nearby souls",
```

Comment:
Maybe add a tooltip specifying the distance meant by "nearby". I also suggest 50 blocks.